docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -0,0 +1,59 @@
+---
+id: app-crash-logs
+title: App Crash Logs
+sidebar_label: App Crash Logs
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+In our latest Beta Program with App Crash Logs feature allows for the automatic capture and display of crash data during both live and automated testing on Sauce Labs' Real Device Cloud (RDC) platform, eliminating the need for integrating an SDK or connecting crash reporting tools to our system.
+
+We're happy to offer a centralized view and a wide range of app diagnostic signals to help teams speed up their debugging process and reduce the time it takes to resolve issues. Our goal is to create faster feedback loops for developers so that they can identify risks and ensure coverage early on in the development cycle. With our new capability, teams can prioritize errors, crashes, and test failures with complete data and context, focusing on the ones that matter the most.
+
+Our tool is capable of detecting and recording fatal errors (crashes) that occur during testing of iOS and Android apps. The call stack details are then stored in a crash.json file along with the test results. Additionally, all non-fatal errors and warnings originating from the native application are also made visible through our interface. This allows for easy debugging and integration with other Mobile App Diagnostics components, allowing you to identify and resolve issues more efficiently.
+
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+- A native Android, iOS, or iPadOS mobile app
+- Make sure instrumentation is enabled, no other configuration is required.
```

Comment:
I think this is the capability that enables instrumentation:  
https://docs.saucelabs.com/dev/test-configuration-options/#networkcapture
Can you confirm and add a link to it here?
```suggestion
- Make sure instrumentation is enabled, no other configuration is required.
```

Reply:
that is not correct. we need a new subdoc and reference outside of this MR for instrumentation. it is just a flag now in app settings page